docs/items/items.md

```diff
@@ -30,4 +30,33 @@ Registering an Item
 
 Items must be [registered][registering] to function.
 
-[registering]: ../concepts/registries.md#registering-things
+Coloring an Item
+----------------
+
+There are items Minecraft prefers to render by coloring one single texture. Leather helmets, spawn eggs, potions and other.
+
+### Item Color Handler
+
+Item color handler is just an instance of `IItemColor`. The interface contains only one method:
```

Comment:
I would reword this to match my suggestion for the block one:

"An item color handler is an instance of `IItemColor`, which adds the `getColorFromItemStack` method as described below."